public/app/features/variables/pickers/shared/VariableLink.tsx

```diff
@@ -36,15 +41,16 @@ export const VariableLink: FC<Props> = ({ loading, onClick: propsOnClick, text,
   }
 
   return (
-    <a
+    <button
       onClick={onClick}
       className={styles.container}
-      aria-label={selectors.pages.Dashboard.SubMenu.submenuItemValueDropDownValueLinkTexts(`${text}`)}
+      data-testid={selectors.pages.Dashboard.SubMenu.submenuItemValueDropDownValueLinkTexts(`${text}`)}
       title={text}
+      id={id}
     >
       <VariableLinkText text={text} />
-      <Icon name="angle-down" size="sm" />
-    </a>
+      <Icon aria-hidden name="angle-down" size="sm" />
+    </button>
```

Comment:
Great work on the a11y. A newb question, as I was trying this on on Mac OS using VoiceOver all the drop downs where announced as buttons. Can we use any `aria-` tag to better describe that this is a drop down and not a button? Or is button the correct choice here?

Reply:
nitpick: Furthermore, you can tab to the drop down options but the options aren't announced when you hover over them.

Reply:
This was the most suitable solution for the moment. I can add aria-expanded and aria-haspopup. Long term we should consider replacing this component with something from Grafana UI (Select?). Mostly because of DRY and to make it more coherent with the design system.